tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- block:
```

Comment:
Why are you using a block for this? There is no condition or error handler attached to the first block.

Reply:
thx